plugins/github/app.mli

```diff
@@ -5,6 +5,9 @@ val input_installation_webhook : unit -> unit
 
 type t
 
+val get_app_id : t -> string
```

Comment:
Needed access to the `app_id` to narrow down the `list_check_runs_for_ref` query against GH.

Reply:
Perhaps `Api.t` needs to change to record the app ID. Since the checks API only works for apps, it might need a type parameter to indicate that (e.g. `app t` vs `oauth t`).

Reply:
Not needed anymore, removing.